src/mqt/predictor/rl/helper.py

```diff
@@ -160,7 +161,9 @@ def get_actions_layout() -> list[dict[str, Any]]:
         {
             "name": "SabreLayout",
             "transpile_pass": lambda c: [
-                SabreLayout(coupling_map=CouplingMap(c)),
+                SabreLayout(coupling_map=CouplingMap(c), skip_routing=True)
+                if metadata.version("qiskit") >= SABRELAYOUT_BREAKING_CHANGE_VERSION
+                else SabreLayout(coupling_map=CouplingMap(c)),
```

Comment:
For the future, we'll have to decide whether we want to keep adding special cases such as here or wether we want to only support fairly recent versions of Qiskit. Since we essentially pin the "pytket-qiskit" package in MQTBench, the range of supported Qiskit versions will be limited anyways. Once a feature is added to MQT Bench that is needed here, the minimum Qiskit version is automatically raised. In the case here, this distinction becomes obsolete once we raise the lower version cap of mqt.bench to at least 0.3.4.

Reply:
Just pushed the changes.